migrations/migration1.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE app ADD COLUMN key varchar(50) DEFAULT md5(random()::text);
```

Comment:
Is MD5 used to hash the app key? It is not cryptographically secure (https://en.wikipedia.org/wiki/MD5), but I don't know if any of the built-in hash function is enough for this application (https://www.postgresql.org/docs/11/functions-binarystring.html). See https://stackoverflow.com/questions/11624372/best-practice-for-hashing-passwords-sha256-or-sha512

Reply:
Now that I think about it probably we can set the default to null, since the idea is to create entries from admin, with an endpoint, so we will use bcrypt 👍